servers/audio_server.cpp

```diff
@@ -1373,7 +1373,7 @@ void AudioServer::init_channels_and_buffers() {
 }
 
 void AudioServer::init() {
-	channel_disable_threshold_db = GLOBAL_DEF_RST("audio/buses/channel_disable_threshold_db", -60.0);
+	channel_disable_threshold_db = GLOBAL_DEF_RST(PropertyInfo(Variant::FLOAT, "audio/buses/channel_disable_threshold_db", PROPERTY_HINT_RANGE, "-100,0,0.01,or_less"), -60.0);
```

Comment:
I want to note that even though **AudioStreamPlayer**'s property is called `volume_db` the suffix is still specified.
```suggestion
	channel_disable_threshold_db = GLOBAL_DEF_RST(PropertyInfo(Variant::FLOAT, "audio/buses/channel_disable_threshold_db", PROPERTY_HINT_RANGE, "-100,0,0.01,or_less,suffix:dB"), -60.0);
```